models/yang/sonic/sonic-mirror-session.yang

```diff
@@ -0,0 +1,60 @@
+module sonic-mirror-session {
+	namespace "http://github.com/Azure/sonic-mirror-session";
+	prefix sms;
+
+	import ietf-inet-types {
+		prefix inet;
+	}
+
+	organization
+		"SONiC";
+
+	contact
+		"SONiC";
+
+	description
+		"SONIC MIRROR SESSION";
+
+	revision 2019-05-15 {
+		description
+			"Initial revision.";
+	}
+
+	container sonic-mirror-session {
+
+		container MIRROR_SESSION {
+
+			list MIRROR_SESSION_LIST {
+				key "name";
```

Comment:
Should we be case insensitive ?
"AB" == "ab" == "Ab" == "aB"

Reply:
Are you talking about ACL name value here ?

Reply:
**Resolved**